backend/src/main/resources/application.properties

```diff
@@ -8,3 +8,5 @@ spring.jpa.hibernate.ddl-auto=update
 spring.jpa.show-sql=true
 
 springdoc.swagger-ui.path=/api
+spring.jackson.date-format=yyyy-MM-dd HH:mm
+spring.jackson.time-zone=Europe/Zagreb
```

Comment:
note: your application might be running simultaneously in different DCs around the world, which could cause great issues :)